testNgExcludeClassAndParameters.xml

```diff
@@ -0,0 +1,15 @@
+<suite name="Excluding Classes and Parameters Suite">
+    <test name="Test with Excluding Classes and Parameters">
+        <classes>
+            <class name="RealItemTest"/>
+            <class name="VirtualItemTest"/>
+            <class name="JsonParserTests"/>
+            <methods>
+                <exclude name=".*"/>
```

Comment:
try to exclude not all methods, but only one by name

Reply:
Done